pkg/ssh/config.go

```diff
@@ -86,14 +87,10 @@ func addHost(path, host, user, context, workspace, command string, gpgagent bool
 	return strings.Join(newLines, "\n"), nil
 }
 
-func GetUser(workspace string) (string, error) {
-	sshConfigPath, err := getSSHConfig()
-	if err != nil {
-		return "", err
-	}
-
+func GetUser(workspace provider.Workspace) (string, error) {
```

Comment:
nit: could you do `workspaceID, sshConfigPath string) like in the other functions please?

Reply:
fixed